platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/FilteringProvider.java

```diff
@@ -77,8 +76,7 @@ protected Value<? extends T> calculateOwnValue(ValueConsumer consumer) {
         }
     }
 
-    @NonNull
```

Comment:
ide suggest to remove this

Reply:
:x: Please don't, it's not related to the change